test/jdk/javax/swing/JTabbedPane/7024235/Test7024235.java

```diff
@@ -105,6 +105,9 @@ private void test(String step) {
         this.passed = true;
         for (int index = 0; index < this.pane.getTabCount(); index++) {
             Rectangle bounds = this.pane.getBoundsAt(index);
+            if (bounds == null) {
```

Comment:
The Aqua L&F uses scroll for the tabs so some of them may be invisible, and then null is returned as specified by the 
`getBoundsAt(index)`

Reply:
Looks ok but it will be good to have a mach5 job listed in JBS testing this modified test on all platforms.

Reply:
> Looks ok but it will be good to have a mach5 job listed in JBS testing this modified test on all platforms.

done

Reply:
One thing...I think it will be good to replace throwing Error with RuntimeException in the test

Reply:
It does not matter which exception or error we will throw, they are treated the same.